react/ReviewForm.tsx

```diff
@@ -447,7 +447,7 @@ export const ReviewForm: FC<InjectedIntlProps & Props> = ({
                 <Input
                   label={intl.formatMessage(messages.emailLabel)}
                   size="large"
-                  value={state.shopperId}
+                  value={state.shopperId || ''}
```

Comment:
Why not set initial state to an empty string on line 85?

Reply:
Done 👍